src/Translatable/Locales.php

```diff
@@ -16,7 +20,7 @@ class Locales implements Arrayable, ArrayAccess
     protected $config;
 
     /**
-     * @var array
+     * @var array<string>
```

Comment:
that's not correct - it would be `array<string, string>` as the locale is key and value.